server/src/v2/routes/secret/submit.js

```diff
@@ -50,9 +50,8 @@ export async function secretSubmit(req, res, next) {
       return result
     })
 
+    var transaction = db.addSecret({
```

Comment:
This change needs to be revereted. What you have done here is include a variable assignment inside of an object.

Reply:
My mistake, must have accidentally hit a combination that moved it, although I also don't recall modifying that portion of the file, so might be worth checking to see if the issue also exists upstream.